src/Expression/ForClasses/HaveNameEndingWith.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+
+namespace Arkitect\Expression\ForClasses;
+
+use Arkitect\Analyzer\ClassDescription;
+use Arkitect\Analyzer\FullyQualifiedClassName;
+use Arkitect\Expression\Description;
+use Arkitect\Expression\Expression;
+use Arkitect\Expression\PositiveDescription;
+use Arkitect\Rules\Violation;
+use Arkitect\Rules\Violations;
+
+class HaveNameEndingWith implements Expression
+{
+    /** @var string */
+    private $name;
+
+    public function __construct(string $name)
+    {
+        $this->name = $name;
+    }
+
+    public function describe(ClassDescription $theClass): Description
+    {
+        return new PositiveDescription("should have a name that ends with {$this->name}");
+    }
+
+    public function evaluate(ClassDescription $theClass, Violations $violations): void
+    {
+        $fqcn = FullyQualifiedClassName::fromString($theClass->getFQCN());
+
+        $sub_str = substr($fqcn->className(), (-1 * strlen($this->name)));
```

Comment:
Variable naming should be in camelCase :)

Reply:
Apologies for the delay in submitting the requested change. I'll be pushing up the change soon. Thanks